### DISCUSSION TITLE
How to setup themes and icons for GTK applications?

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MINGW-packages#20555] [General]
For example, if I install `mingw-w64-gtk-elementary-theme` and `mingw-w64-elementary-icon-theme`, can I expect the GTK applications to be possible to see and use them?

**Comment by lazka** [comment]
In theory, yes: `GTK_THEME=io.elementary.stylesheet.blueberry gtk3-widget-factory.exe`
